ext/bin/check-command-perf

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+usage() {
+    cat <<EOF
+Usage: [PDB_JAR=JAR] PDBBOX=BOX_DIR $(basename "$0") STOCKPILE_ARCHIVE"
+
+  Unarchive STOCKPILE_ARCHIVE into ${PDBBOX}/var/stockpile/cmd/q and run
+  pdb to time how long it takes to consume the stockpile files. The results
+  of this test can be seen in the console or the results_log file that is
+  created after every run.
+
+  The version of pdb will be the one run via ./pdb which defaults to
+  the uberjar in target/.
+
+EOF
+}
+
+misuse() { usage 1>&2; exit 2; }
+
+jar="${PDB_JAR:-target/puppetdb.jar}"
+
+if test "$#" -lt 0 -o -z "$PDBBOX"; then
+  printf '$PDBBOX variable is not set. Do you have a pdbbox created?\n' 1>&2
+  misuse
+fi
+
+if ! test -e "$jar"; then
+    printf 'Unable to find the puppetdb jar %q; have you run "lein uberjar"?\n' \
+           "$jar" 1>&2
+    misuse
```

Comment:
I don't think it's necessary to print the usage for this specific messages. Or at least add an additional empty line between the error and the usage. Another empty line before the error message would be appreciated.